ais_bench/benchmark/datasets/dapo_math.py

```
@@ -0,0 +1,405 @@
+                    if isinstance(item, dict) and item.get('role') == 'user':
+                        prompt_content = item.get('content', '')
+                        break
+                
+                if prompt_content is None:
+                    # Fallback: use first item's content
+                    if isinstance(prompt_list[0], dict):
+                        prompt_content = prompt_list[0].get('content', '')
+                    else:
+                        prompt_content = str(prompt_list[0])
+                
+                # Extract ground truth from reward_model
+                if 'reward_model' not in row:
+                    raise AISBenchDataContentError(
+                        DSET_CODES.DATA_INVALID_STRUCTURE,
+                        f"Missing 'reward_model' field at row {idx}"
+                    )
+                
+                reward_model = row['reward_model']
+                if not isinstance(reward_model, dict) or 'ground_truth' not in reward_model:
+                    raise AISBenchDataContentError(
+                        DSET_CODES.DATA_INVALID_STRUCTURE,
+                        f"Invalid 'reward_model' format at row {idx}: missing 'ground_truth'"
+                    )
+                
+                ground_truth = str(reward_model['ground_truth'])
+                
+                # Build dataset entry
+                entry = {
+                    'prompt': prompt_content,
+                    'answer': ground_truth,
+                    'data_source': row.get('data_source', 'math_dapo'),
+                    'ability': row.get('ability', 'MATH'),
+                    'extra_info': row.get('extra_info', {}),
+                }
+                
+                dataset.append(entry)
+                
+            except AISBenchDataContentError:
+                raise
+            except Exception as e:
+                logger.debug(f"Unexpected error processing row {idx}: {e}")
+                continue
```

【review】Catching a broad Exception and logging it with logger.debug can silently hide issues with data quality or processing logic, as debug logs are often not visible by default. This could lead to an incomplete or partially incorrect dataset without any warning. It's better to use logger.warning to make these potential issues more visible.
```
            except Exception as e:
                logger.warning(f"Unexpected error processing row {idx}, skipping row: {e}")
                continue
```